sources/rest_api/detector.py

```diff
@@ -29,6 +31,11 @@
 NEXT_PAGE_DICT_KEY_PATTERNS = {"href", "url"}
 
 
+def single_entity_path(path: str) -> bool:
```

Comment:
Should this handle trailing slashes (e.g. `"/users/{user_id}/"`) ? I'll add a couple of tests in a PR for this. Most likely we need some high level default treatment for those.